examples/js/loaders/AMFLoader.js

```diff
@@ -206,6 +206,7 @@ THREE.AMFLoader.prototype = {
 
 				} else if ( matColor.nodeName === 'a' ) {
 
+					color.a = matColor.textContent;
```

Comment:
6 letters? :monkey: 

Removed. :)  There was also a typo in materialid that had broken materials.  Updated.